modules/eb_widget/js/dropzonejs_eb_widget.common.js

```diff
@@ -13,7 +13,7 @@
           var $form = $(item.instance.element).parents('form');
 
           if ($form.hasClass("dropzonejs-disable-submit")) {
-            var $submit = $form.find('[type=submit]');
+            var $submit = $form.find('[type=submit]').not('.tab');
```

Comment:
Could we limit this the other way around? i.e. instead of "All submits but tabs." target "Dropzone submit and nothing else."